packages/rocketchat-message-pin/client/views/pinnedMessages.js

```diff
@@ -0,0 +1,69 @@
+/* globals PinnedMessage */
+Template.pinnedMessages.helpers({
+	hasMessages() {
+		return PinnedMessage.find({
+			rid: this.rid
+		}, {
+			sort: {
+				ts: -1
+			}
+		}).count() > 0;
+	},
+	messages() {
+		return PinnedMessage.find({
+			rid: this.rid
+		}, {
+			sort: {
+				ts: -1
+			}
+		});
+	},
+	message() {
+		return _.extend(this, {
+			customClass: 'pinned'
+		});
+	},
+	hasMore() {
+		return Template.instance().hasMore.get();
+	}
+});
+
+Template.pinnedMessages.onCreated(function() {
+	this.hasMore = new ReactiveVar(true);
+	this.limit = new ReactiveVar(50);
+	return this.autorun(() => {
+		return function() {
+			const data = Template.currentData();
+			return this.subscribe('pinnedMessages', data.rid, this.limit.get(), function() {
```

Comment:
If you use `function` here the `this` inside the function will be referent to the function, as we are trying to access the `this.hasMore` above you should use `=>`